src/Parser/Core/Modules/ManaValues.js

```diff
@@ -26,9 +37,29 @@ class ManaValues extends Analyzer {
             max: max,
             used: manaCost,
           });
+          // The variable 'max' is constant but can differentiate by racial/items.
+          this.baseMana = max;
```

Comment:
I'm unsure if calling the *max* amount of mana "baseMana" is going to always be right. How about actually naming this `max`?